Makefile

```diff
@@ -29,18 +29,18 @@ ifneq ($(DIRTY),)
 VERSION := $(VERSION)+dirty
 endif
 
-GO_LD_FLAGS = -ldflags "-X main.VERSION=$(VERSION)"
+GO_LD_FLAGS = -X main.VERSION=$(VERSION)
 
 all: controller kubeseal
 
 generate: $(GO_FILES)
 	$(GO) generate $(GO_PACKAGES)
 
 controller: $(GO_FILES)
-	$(GO) build -o $@ $(GO_FLAGS) ./cmd/controller
+	$(GO) build -o $@ $(GO_FLAGS) -ldflags "$(GO_LD_FLAGS)" ./cmd/controller
 
 kubeseal: $(GO_FILES)
-	$(GO) build -o $@ $(GO_FLAGS) $(GO_LD_FLAGS) ./cmd/kubeseal
+	$(GO) build -o $@ $(GO_FLAGS) -ldflags "$(GO_LD_FLAGS)" ./cmd/kubeseal
 
 %-static: $(GO_FILES)
 	CGO_ENABLED=0 $(GO) build -o $@ -installsuffix cgo $(GO_FLAGS) ./cmd/$*
```

Comment:
Need ldflags here too..  (This Makefile could do with some refactoring..)
In many ways, this is the only controller target that matters, since it's the one that goes into the docker image.

(:thinking: Perhaps this indicates the integration test should instead exec the controller binary in the k8s container, or verify the correctly versioned log entry exists?)

Reply:
@anguslees hmmm yes I see. It seems like I've tested the non static version and then assumed it's always the same which it's not. I suppose we could add some docker run commands into the integration suite, going to take a look at that.